### DISCUSSION TITLE
PrimeVue form component + validation

### DISCUSSION THREAD

**Discussion by dinacuatro** [primefaces/community#2561] [PrimeVue]
For form level attributes (like `disabled` for example) it would be good to have a form component, is it part of the roadmap?
Also need to know what about form validation, is it going to be soon part of PrimeVue suite? In case it is, when?

**Comment by j0rgedev** [comment]
It is actually on the [roadmap](https://primevue.org/roadmap/). It says Q3 so it might be under development. As a workaround, you could try this library [formkit-primevue](https://github.com/sfxcode/formkit-primevue)
